src/main/java/Duke.java

```diff
@@ -30,17 +32,15 @@ public static void main(String[] args) {
         System.out.print("Enter command: ");
 
         while (!currProgram.getCanTerminateHal()) {
```

Comment:
getCanTerminateHal does not comply with coding standard i think?

Reply:
canTerminateHal was the boolean though. So i guess since getCanTerminateHal is a getter, it should be fine